e-commerce-project/src/app/components/login-page/login-page.component.ts

```diff
@@ -7,13 +8,20 @@ import { Component, OnInit } from '@angular/core';
 })
 export class LoginPageComponent implements OnInit {
 
-  toSubmit(userName, password){
-    console.log("Username is '" + userName + "' and password is '" + password + "'.");
-  }
+  loginForm: FormGroup;
 
-  constructor() { }
+  constructor(public formBuilder: FormBuilder) { 
+  	this.loginForm = formBuilder.group({
+  		userName: [null],
+      	password: [null, Validators.minLength(8)]
```

Comment:
we have two arguments:
 * first is initial value which we set to null
 * second argument is validation (we use helper class `Validators` with method `minLenth()`